### PR TITLE
Feature/subcontent styling

### PR DESCRIPTION
**Pull request by Snugug**
Relates to #224

Styles Aside, Color, Detail, and Example subcontent

### CODE REVIEW

**Comment by ryanbrownhill** [comment]
:+1: :boy: says :green_apple:

**Comment by poofichu** [comment]
:+1: :icecream:

File: language/02-guidelines/01-style/color/index.md [hunk 1]
```diff
@@ -14,13 +14,19 @@ variables:
...
+          
+          Consider users who are sensitive to brightness and high contrast colors by using a neutral value as a background color instead of pure white.
       dark:
         title: "Example: Dark Palette"
         visualDisplay: /images/color/dark-palette.svg
-        description: "Primary: Blue 30, Secondary: Orange 30, Accent: Teal 30"
-        caption: "Consider users who are sensitive to brightness and high contrast colors by using a Gray value as a background color instead of pure black."
+        description: >
+          Primary: Blue 30, Secondary: Orange 30, Accent: Teal 30
+
```
Comment (bobbinrobyn) [inline language/02-guidelines/01-style/color/index.md]: This giant space is on purpose, right?

Comment (Snugug) [inline language/02-guidelines/01-style/color/index.md]: Yes. That creates 2 paragraphs

**Comment by bobbinrobyn** [comment]
:+1:

**Pull request merged**